ui/app/AppLayouts/Chat/CommunityColumn.qml

```diff
@@ -35,7 +35,9 @@ Item {
             qsTr("%1 Members").arg(chatsModel.communities.activeCommunity.nbMembers)
         chatInfoButton.image.source: chatsModel.communities.activeCommunity.thumbnailImage
         chatInfoButton.icon.color: chatsModel.communities.activeCommunity.communityColor
-        chatInfoButton.onClicked: communityProfilePopup.open()
+        chatInfoButton.onClicked: openPopup(communityProfilePopup, {
+            community: chatsModel.communities.activeCommunity
```

Comment:
FYI, from what I learned from @emizzle , doing something like this doesn't bind the data, so if something changes in `activeCommunity`, the popup's data wouldn't update. Is that a problem? If the popup is meant only for info, then close, I feel like it's ok.

Reply:
Ah, that's a good point.. I guess if we want to be super sure, we should go the safe route.

On the other hand, how much can possibly change in the community when this popup was opened? Maybe its description, image and members, but I think it's fine if that isn't updated in realtime while the popup is open.

Happy to change tho, of there's strong opinions!

Reply:
Yeah, I don't think it should  update much, so I think for now it's fine as is.